dll/shellext/deskadp/lang/pt-PT.rc

```diff
@@ -30,7 +37,7 @@ FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
     DEFPUSHBUTTON "OK", IDOK, 112, 115, 50, 15
     PUSHBUTTON "Cancelar", IDCANCEL, 167, 115, 50, 15
-    GROUPBOX "Lista de modos válidos", -1, 6, 7, 212, 98
+    GROUPBOX "&Lista de modos válidos", -1, 6, 7, 212, 98
```

Comment:
What's the users benefit from having an accelerator at a groupbox?

Reply:
I just followed the WinSrv2003 implementation!

![VirtualBox_WinSrv2003_ListModes](https://github.com/reactos/reactos/assets/49964946/93afee97-0a83-40db-aa2d-14b2e7506425)

Reply:
I didn't understand your message.
Do you want me to remove the accelerator key from the "List of Valid Modes" group?

Reply:
The accelerator is OK.